apps/HelloHexagon/Makefile

```diff
@@ -29,7 +29,7 @@ pipeline_hvx64-%.o: pipeline
 	HL_TARGET=$*-hvx_64 ./pipeline pipeline_hvx64-$* pipeline_hvx64
 
 pipeline_hvx128-%.o: pipeline
-	HL_TARGET=$*-hvx_128 ./pipeline pipeline_hvx128-$* pipeline_hvx128
+	HL_TARGET=$*-profile-hvx_128 ./pipeline pipeline_hvx128-$* pipeline_hvx128
```

Comment:
I've been using it to roughly eyeball performance regressions, but it's not a big deal. I think @ronlieb might be using this as a performance indicator.